README.md

```diff
@@ -3,38 +3,42 @@
 
 # vSwarm - Serverless Benchmarking Suite
 
-Welcome! vSwarm is a collection of ready-to-run serverless benchmarks, each 
+Welcome! vSwarm is a collection of ready-to-run serverless benchmarks, each
 typically consisting of a number of interconnected serverless functions, and with a general
 focus on realistic data-intensive workloads.
 
-This suite is a turnkey and fully tested solution meant to used in conjunction with 
+This suite is part of the [vHive Ecosystem](https://vhive-serverless.github.io/). Its a turnkey and fully
+tested solution meant to used in conjunction with
 [vHive](https://github.com/ease-lab/vhive), and is compatible with all technologies that it supports,
 namely, containers, Firecracker and gVisor microVMs. The majority of benchmarks support distributed
-tracing with [Zipkin](https://zipkin.io/) which traces both the infra components and the user 
+tracing with [Zipkin](https://zipkin.io/) which traces both the infra components and the user
 functions.
+Additionally, [several](./benchmarks/README.md#standalone-functions-benchmark-summary) of the benchmarks are compatible with
+[vSwarm-u](https://github.com/ease-lab/vSwarm-u), a framework that integrates the benchmarks with [gem5](https://www.gem5.org/),
```

Comment:
Still old here

Reply:
Good catch

Reply:
Ah remembered I did it because it was the first. But now everything is moved so now everything is clean